src/middlewares/Validate.js

```diff
@@ -67,5 +48,29 @@ class ValidateUser {
       check('startDate', 'date should be validate like in this format(YYYY-DD--MM)').isBefore(startDates)
     ];
   }
+
+  /**
+* Validate input
+* @static
+* @returns {object} errors
+*/
+  static profileUpdateRules() {
+    const d = new Date();
+    const year = d.getFullYear();
+    const month = d.getMonth();
+    const day = d.getDate();
+    const minAge = new Date(year - 16, month, day).toDateString();
+    return [
+      oneOf([
+        check('gender').isIn(['male', 'female']),
+      ], 'Gender should be either male or female'),
+      check('birthDate', 'Invalid Date of Birth(format: YYYY-MM-DD && Make sure you are above 16)').isBefore(minAge),
+      check('address', 'Address should be specified').isAlphanumeric(),
+      check('imageURL', 'PRofile image URL should be valid'),
```

Comment:
@victorkarangwa4 I think one can easily do this by customizing the check which is possible with express validator by even adding the isImage middleware here and have something like this 

```
check('imageURL', 'Profile image URL should be valid').custom((value, { req }) => {
    if (!isImageUrl(req.body.imageURL)) {
 return Customize.errorMessage(req, res, 'Invalid image url', 400);
 }
  });
```
Try this and see if it works just fine